spark/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala

```diff
@@ -310,6 +310,9 @@ class DeltaSqlAstBuilder extends DeltaSqlBaseBaseVisitor[AnyRef] {
     VacuumTableCommand(
       Option(ctx.path).map(string),
       Option(ctx.table).map(visitTableIdentifier),
+      Option(ctx.reservoirPath).map(string),
```

Comment:
Can we use named arguments here? Because we have a number of same-type parameters here it's very easy to get the order wrong.